### PR TITLE
Mass actions rework

### PR DESCRIPTION
**Pull request by AEnterprise**
**What does this PR add/fix/improve/...**
more mass actions and a full rework how it's handled internally
want some extra eyes in case this breaks as these commands are usually only used when it's urgent so want to avoid them breaking cause i missed something

### CODE REVIEW

File: GearBot/Util/Actions.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        return False, Translator.translate(f'{action}_unable', ctx.guild.id, user=Utils.clean_user(user))
+
+    if ((ctx.author != user and ctx.author.top_role > user.top_role) or (
+            ctx.guild.owner == ctx.author)) and user != ctx.guild.owner and user != ctx.bot.user and ctx.author != user:
+        return True, None
+    if user.bot and not action_bot:
+        return False, Translator.translate(f"cant_{action}_bot", ctx.guild.id, user=user)
+
+    else:
+        return False, Translator.translate(f'{action}_not_allowed', ctx.guild.id, user=user)
```
Comment (trilleplay) [inline GearBot/Util/Actions.py]: This `else:` is not needed.

File: lang/en_US.json [hunk 2]
```diff
@@ -772,5 +772,7 @@
   "verification_help": "Changes the Verification Level",
   "mod_nickname_help": "Base command for nickname.",
   "mod_nickname_add_help": "Adds or edits a user's nickname.",
-  "mod_nickname_remove_help": "Removes the user's nickname."
+  "mod_nickname_remove_help": "Removes the user's nickname.",
```
Comment (trilleplay) [inline lang/en_US.json]: It's users not user's

File: GearBot/Util/Actions.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        if require_on_server:
+            message = Translator.translate('user_not_on_server', ctx.guild.id)
+            if send_message:
+                await ctx.send(f"{Emoji.get_chat_emoji('NO')} {message}")
+            return False, message
+        else:
+            user = ctx.bot.get_user(target)
+    if user is None:
+        return False, "Unknown user"
+    else:
```
Comment (trilleplay) [inline GearBot/Util/Actions.py]: You're returning here, the `else` argument is not needed.

File: lang/en_US.json (hunk 2 above)
Comment (AEnterprise) [reply lang/en_US.json]: this is correct afaik: https://english.stackexchange.com/questions/36657/user-s-guide-vs-users-guide

Comment (trilleplay) [reply lang/en_US.json]: Resolved :+1:

**Pull request merged**